input4MIPs_source_id.json

```diff
@@ -2869,6 +2869,22 @@
             "version":"20181127",
             "xlink":"http://cera-www.dkrz.de/WDCC/meta/CMIP6/input4MIPs.CMIP6.AerChemMIP.UoM.UoM-AIM-ssp370-lowNTCF-1-2-1.atmos.yr.mole_fraction_of_sf6_in_air.gr1-GMNHSH.20181127.json|Citation|citation"
         },
+        "CR-DECK-0-1-0":{
+            "contact":"zebedee.nicholls@climate-resource.com",
```

Comment:
@durack1 which fields are compulsory and which are optional?

Is it worth adding an automated check (GitHub CI) that makes sure that what users add here is valid/correct?